docs/contributing.rst

```diff
@@ -69,6 +69,45 @@ found in the `virtualenv docs`_.
 
 .. _`virtualenv docs`: https://virtualenv.pypa.io
 
+Running certbot against a locally deployed Pebble CA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By following the instructions above, you can execute Certbot against a local instance
```

Comment:
I think this is what you mean, but you should double check.

I also added the word "manually" to differentiate it from the automated integration tests.

```suggestion
By following the instructions below, you can manually execute Certbot against a local instance
```

Reply:
In fact I wanted to refer to the previous section to ensure that the user is in a properly set up venv, because you need that before calling the commands exposed by certbot-ci. I improved the clarity of this paragraph.